tests/features/joinup_discussion/discussion.invite.feature

```diff
@@ -194,16 +194,15 @@ Feature: Invite members to subscribe to discussions
     # Initially there should not be any subscriptions.
     And the "Concerned about dissolved gases?" discussion should have 0 subscribers
     Given I am logged in as "Glory Ruskin"
-    When I accept the invitation for the "Concerned about dissolved gases?" discussion
+    When I accept the invitation for the "Concerned about dissolved gases?" discussion content
     Then I should see the heading "Concerned about dissolved gases?"
     And I should see the success message "You have been subscribed to this discussion."
     And the "Concerned about dissolved gases?" discussion should have 1 subscriber
 
     # Try to accept the invitation a second time. An appropriate message should
     # be shown.
-    When I accept the invitation for the "Concerned about dissolved gases?" discussion
-    Then I should see the heading "Concerned about dissolved gases?"
-    And I should see the success message "You were already subscribed to this discussion."
+    When I accept the invitation for the "Concerned about dissolved gases?" discussion content
+    Then I should see the heading "Access denied"
```

Comment:
Indeed, I can keep it. I could not find the ticket that requested it.

Reply:
I had to do a couple of more changes for this. Up to now, we were re-accepting/rejecting the invitation and then issuing an event. This is not correct as the invitation is not supposed to have action on it more than once. 
I created a new event for not pending invitations so that it is more clear what message to print.

Reply:
> Indeed, I can keep it. I could not find the ticket that requested it.

It's in one of the comments of ISAICP-4090: https://citnet.tech.ec.europa.eu/CITnet/jira/browse/ISAICP-4090?focusedCommentId=2538324&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2538324